de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/server/rpl/imp/RabbitMqListener.java

```diff
@@ -0,0 +1,306 @@
+package org.adempiere.server.rpl.imp;
+
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2015 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+import de.metas.logging.LogManager;
+import de.metas.util.Services;
+import org.adempiere.ad.trx.api.ITrxManager;
+import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.process.rpl.XMLHelper;
+import org.adempiere.server.rpl.IReplicationProcessor;
+import org.adempiere.server.rpl.api.IIMPProcessorBL;
+import org.adempiere.server.rpl.api.IImportHelper;
+import org.adempiere.server.rpl.exceptions.ReplicationException;
+import org.adempiere.util.lang.Mutable;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitAdmin;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
```

Comment:
```suggestion
```

Reply:
i would also avoid "*" imports.
I think IDEs are good enough to make it easy to precisely import what u need.